### PR TITLE
chore: bump akash version to v0.22.7

### PR DESCRIPTION
**Pull request by troian**

### CODE REVIEW

**Comment by troian** [comment]
@yaruwangway mind approving ci

File: akash/chain.json [hunk 1]
```diff
@@ -27,27 +27,25 @@
...
       "genesis_url": "https://raw.githubusercontent.com/akash-network/net/master/mainnet/genesis.json"
     },
     "versions": [
       {
-        "name": "v0.22.0",
-        "recommended_version": "v0.22.0",
+        "name": "v0.22.7",
+        "recommended_version": "v0.22.7",
         "compatible_versions": [
-          "v0.20.0"
```
Comment (JeremyParish69) [inline akash/chain.json:46]: Can we please retain and expand the compatible_versions arrays?

Comment (troian) [reply akash/chain.json:46]: at this moment there `v0.22.7` is the only version that is compatible

Comment (JeremyParish69) [reply akash/chain.json:46]: oh, then, can we please create a new version object (so one for v.0.22.0, and another for v.0.22.7), and also include the version name in its own compatible versions array.

**Comment by dylanschultzie** [comment]
closing this due to it being outdated

**Pull request closed without merging**